src/pipecat/transports/services/daily.py

```diff
@@ -819,6 +819,7 @@ def __init__(
         # these handlers.
         self._register_event_handler("on_joined")
         self._register_event_handler("on_left")
+        self._register_event_handler("on_error")
```

Comment:
I believe we still miss

```
await self._call_event_handler("on_error", error)
```

from the `_on_error(self, error)` function. I actually added a `TODO` note there. Because I was thinking we should probably do:

```
    async def _on_error(self, error):
        await self._call_event_handler("on_error", error)
        await self.push_error(ErrorFrame(error))
```